db/seeds/06_review_seeds.rb

```diff
@@ -0,0 +1,65 @@
+user_id = User.all.sample.id
+
+reviews_data = [
+  [
+    {rating: 5, comment: "Showed me the most interesting route", user_id: user_id},
+    {rating: 3, comment: "Friendly, but not very responsive over the app", user_id: user_id}
+  ],
+  [
+    {rating: 4, comment: "Good belayer", user_id: user_id},
+    {rating: 3, comment: "Nice, but always late...", user_id: user_id}
+  ],
+  [
+    {rating: 5, comment: "Great technique and good coach", user_id: user_id},
+    {rating: 3, comment: "Special kind of humour", user_id: user_id}
+  ],
+  [
+    {rating: 5, comment: "Best pancakes!", user_id: user_id},
+    {rating: 5, comment: "Responsible and trustworthy", user_id: user_id},
+  ],
+  [
+    {rating: 5, comment: "Nothing to complain about", user_id: user_id},
+    {rating: 1, comment: "Didn't show up", user_id: user_id},
+  ],
+  [
+    {rating: 5, comment: "Knows the best cafe in town", user_id: user_id},
+    {rating: 5, comment: "Picked me up by car", user_id: user_id},
+  ],
+  [
+    {rating: 5, comment: "Great guest bed", user_id: user_id},
+    {rating: 5, comment: "Great host", user_id: user_id},
+  ],
+  [
+    {rating: 2, comment: "Didn't tell me about the cat in advance", user_id: user_id},
+    {rating: 5, comment: "Friendly cat", user_id: user_id},
+  ],
+  [
+    {rating: 5, comment: "Knows how to cheer you up", user_id: user_id},
+    {rating: 5, comment: "Borrowed me some gear", user_id: user_id},
+  ],
+  [
+    {rating: 2, comment: "We didn't match very well", user_id: user_id},
+    {rating: 5, comment: "Direct and efficient communicator", user_id: user_id},
+  ],
+  [
+    {rating: 1, comment: "No previous climbing experience whatsoever", user_id: user_id},
+    {rating: 1, comment: "Kept on calling me after our trip", user_id: user_id},
+  ]
+]
+
+def seed_reviews(reviews, n)
+  Profile.all.each_with_index do |profile, index|
+    if Profile.find(index + n)
+      r = Review.new(reviews[0])
```

Comment:
<img width="1051" alt="Screenshot 2022-09-03 at 17 16 29" src="https://user-images.githubusercontent.com/105738004/188277212-6a444317-341d-482a-80aa-daed3cad6fe7.png">

reviews[0] is an array of two hashes:
`reviews_data = [
  [
    {rating: 5, comment: "Showed me the most interesting route", user_id: user_id},
    {rating: 3, comment: "Friendly, but not very responsive over the app", user_id: user_id}
  ],`

you need to pass a hash

Reply:
Fixed it!